gobblin-azkaban/src/main/java/gobblin/azkaban/AzkabanJobLauncher.java

```diff
@@ -49,7 +58,7 @@
  *
  * @author Yinan Li
  */
-public class AzkabanJobLauncher extends AbstractJob {
+public class AzkabanJobLauncher extends AbstractJob implements ApplicationLauncher, JobLauncher {
```

Comment:
Its mostly just to declare that `AzkabanJobLauncher` is a `JobLauncher` - that is, it is capable of launching jobs. In this specific scenario there wouldn't really be case where someone would directly call `launchJob`, I think it just makes the code easier to understand.